### PR TITLE
Add JustMockLite to additional resources

### LINKED ISSUE #19223: Adding a third-party open source project to the MVC unit testing article
Hi, guys! My name is Mihail Vladov and I am the engineering manager of [JustMockLite ](https://github.com/telerik/JustMockLite )which is an open-source mocking framework. My team and I are part of [Telerik ](https://www.telerik.com/) which is a long time partner with Microsoft.
I wanted to ask you if it is possible to add JustMockLite to the additional resources in the following article?  
[https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/testing?view=aspnetcore-3.1](https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/testing?view=aspnetcore-3.1)

I am willing to submit а pull request with all the necessary changes.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b5cce372-24e2-3489-cb19-c85d934e5d3b
* Version Independent ID: 9d12b19f-a94e-4df4-357b-3f3b93696eb6
* Content: [Test controller logic in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/mvc/controllers/testing?view=aspnetcore-3.1)
* Content Source: [aspnetcore/mvc/controllers/testing.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/mvc/controllers/testing.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-mvc**
* GitHub Login: @ardalis
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by mihail-vladov**
Fixes #19223

As discussed in the issue, I am adding JustMockLite to links in the additional resources section.
Please have in mind that I also changed the "Razor Pages unit tests in ASP.NET Core" article to include the same link in the additional resources section.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=19284) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=19284) before we can accept your contribution.<br/><br/>:x: mihail-vladov [sign now](https://cla.dotnetfoundation.org/dotnet/AspNetCore.Docs?pullRequest=19284)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/AspNetCore.Docs?pullRequest=19284) it.</sub>

File: aspnetcore/mvc/controllers/testing.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: Learn how to test controller logic in ASP.NET Core with Moq and xUn
 monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 11/07/2019
+ms.date: 22/07/2020
```
Comment (guardrex) [inline aspnetcore/mvc/controllers/testing.md:8]: Change the date format to `7/22/2020` ...

```suggestion
ms.date: 7/22/2020
```

File: aspnetcore/mvc/controllers/testing.md [hunk 2]
```diff
@@ -344,4 +344,5 @@ For a valid session `id`, the final test confirms that:
 * <xref:test/integration-tests>
 * [Create and run unit tests with Visual Studio](/visualstudio/test/unit-test-your-code)
 * [MyTested.AspNetCore.Mvc - Fluent Testing Library for ASP.NET Core MVC](https://github.com/ivaylokenov/MyTested.AspNetCore.Mvc): Strongly-typed unit testing library, providing a fluent interface for testing MVC and web API apps. (*Not maintained or supported by Microsoft.*)
+* [JustMockLite](https://github.com/telerik/JustMockLite): The most powerful free mocking framework available for .NET developers. (*Not maintained or supported by Microsoft.*)
```
Comment (guardrex) [inline aspnetcore/mvc/controllers/testing.md:347]: Remove the marketing language ...

```suggestion
* [JustMockLite](https://github.com/telerik/JustMockLite): A free mocking framework available for .NET developers. (*Not maintained or supported by Microsoft.*)
```

File: aspnetcore/test/razor-pages-tests.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ description: Learn how to create unit tests for Razor Pages apps.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
 ms.custom: mvc
-ms.date: 08/14/2019
+ms.date: 22/07/2020
```
Comment (guardrex) [inline aspnetcore/test/razor-pages-tests.md:8]:
```suggestion
ms.date: 7/22/2020
```

File: aspnetcore/test/razor-pages-tests.md [hunk 4]
```diff
@@ -376,5 +376,6 @@ Other tests in this group create page model objects that include the <xref:Micro
 * [Getting started with xUnit.net: Using .NET Core with the .NET SDK command line](https://xunit.github.io/docs/getting-started-dotnet-core)
 * [Moq](https://github.com/moq/moq4)
 * [Moq Quickstart](https://github.com/Moq/moq4/wiki/Quickstart)
+* [JustMockLite](https://github.com/telerik/JustMockLite): The most powerful free mocking framework available for .NET developers. (*Not maintained or supported by Microsoft.*)
```
Comment (guardrex) [inline aspnetcore/test/razor-pages-tests.md:379]:
```suggestion
* [JustMockLite](https://github.com/telerik/JustMockLite): A free mocking framework available for .NET developers. (*Not maintained or supported by Microsoft.*)
```

**Comment by guardrex** [comment]
Thanks ... Rick will be along in a bit for a final sign-off on this.

**Pull request merged**